modules/redhat_manifest.py

```diff
@@ -137,7 +137,6 @@ def create_manifest(module):
     path = "/subscription/consumers"
     data = {'name': module.params['name'],
             'type': "satellite",
-            'owner': module.params['rhsm_owner'],
```

Comment:
@sean797 do you remember why you added this? I get weird errors when I leave this in, both against prod and stage.

```
fatal: [tests]: FAILED! => {"changed": false, "msg": "POST to https://subscription.rhn.stage.redhat.com/subscription/consumers failed, got Runtime Error com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `org.candlepin.dto.api.v1.OwnerDTO` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('11612516')\n at [Source: (org.candlepin.common.filter.TeeHttpServletRequest$1); line: 1, column: 11] (through reference chain: org.candlepin.dto.api.v1.ConsumerDTO[\"owner\"]) at com.fasterxml.jackson.databind.exc.MismatchedInputException.from:63"}
```

It works just fine when I remove the `owner`, so that's what I did for now.

Reply:
No - but since its not in https://github.com/Katello/hammer-cli-csv/blob/51f64ea29fd86affb66db1ab3dde429097a8c8b6/lib/hammer_cli_csv/subscriptions.rb#L187 lets remove it

Reply:
No - but since its not in https://github.com/Katello/hammer-cli-csv/blob/51f64ea29fd86affb66db1ab3dde429097a8c8b6/lib/hammer_cli_csv/subscriptions.rb#L187 lets remove it